alacritty/src/renderer/mod.rs

```diff
@@ -418,6 +418,8 @@ struct InstanceData {
     bg_g: f32,
     bg_b: f32,
     bg_a: f32,
+    // boolean set to true if the glyph already is colored (typically RGBA emojis)
```

Comment:
I feel like it's too long tbh, maybe something like that

```suggestion
    // Colored glyph indicator.
```